css/properties/grid-template-columns.json

```diff
@@ -178,7 +178,8 @@
                 }
               ],
               "ie": {
-                "version_added": false
+                "version_added": "11",
+                "notes": "IE can treat the first (<code>min-content</code>) parameter as the <code>max-content</code> parameter (source: <a href='https://css-tricks.com/css-grid-in-ie-css-grid-and-the-new-autoprefixer/#article-header-id-10'>CSS grid in IE: CSS Grid and the New Autoprefixer</a>)."
```

Comment:
Hi @anneke. Based on the feedback on this PR, I think you can remove this note (leaving only the IE 11 version data). If you could make that change, then we can merge this PR.

Thanks for your help and welcome to BCD!

Reply:
Thanks all! I forgot I made this PR but am very happy with your feedback, thanks! (👋 @rachelandrew) . I’ll update the PR tomorrow.

Reply:
I'm sorry it took me a little longer than I thought, but here you go! Thanks for letting me contribute :)